pkg/server/bootstrap/identity.go

```diff
@@ -282,7 +282,11 @@ func decorateWildcardPathsWithResourceIdentities(urlPath string, ids *identities
 		if len(id) == 0 {
 			return "", fmt.Errorf("identity for %s is unknown", gr)
 		}
-		comps[5] += ":" + id
```

Comment:
Instead of this, we could alternatively put this after line 278 (after we ensure we have at least 6 in comps):
```go
if strings.Contains(comps[5], ":") {
	return urlPath, nil
}
```